### PR TITLE
keep and use crc calculations for rfx tiles to save bandwidth when an app redraws aggressively

### PR DESCRIPTION
**Pull request by jsorg71**

### CODE REVIEW

**Comment by johnarnold** [comment]
Validated manually on a test VM!

**Comment by metalefty** [comment]
No massive

`[2756639.134] rdpDeferredUpdateCallback: reschedule rect_id 823656 rect_id_ack 823655` 

anymore.   Now I got 

`[528408.817] rdpClientConSendPaintRectShmEx: nothing to send`

instead but maybe it is normal. I experience better performance now.

**Comment by jsorg71** [comment]
I think we get remove "nothing to send" too
It's more for debugging

File: module/rdpCapture.c [hunk 1]
```diff
@@ -811,8 +878,13 @@ rdpCapture2(rdpClientCon *clientCon, RegionPtr in_reg, BoxPtr *out_rects,
     RegionRec tile_reg;
     const char *src;
     char *dst;
+    char *crc_dst;
```
Comment (metalefty) [inline module/rdpCapture.c]: use `uint8_t` for `*src`, `*dst`, `*crc_dst`. changed by #90.

**Comment by metalefty** [comment]
Oops, I tried to resolve conflicts via GitHub web but failed... I'll fix it, sorry.

**Comment by metalefty** [comment]
I've created #94. It is able to be merged into devel and works fine as well.
Sorry about the breakage :sob:

**Pull request closed without merging**